Bottom: 75fe80b8f56fda57774019d83f136ec682538f5e
Top:    af36419776a6ea085693be2de887edb28acde20e
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-04 16:53:37 -0500

Refresh of TESTING--VirtualMachine

---
diff --git a/ansible-playbook/group_vars/docker-hosts/repos.yml b/ansible-playbook/group_vars/docker-hosts/repos.yml
index 8917e7ce..ae3590a2 100644
--- a/ansible-playbook/group_vars/docker-hosts/repos.yml
+++ b/ansible-playbook/group_vars/docker-hosts/repos.yml
@@ -7,3 +7,5 @@ wiki_image_repo: https://github.com/AndrewCooper/docker-mediawiki.git
 mantisbt_image_repo: https://github.com/AndrewCooper/docker-mantisbt.git
 svn_image_repo: https://github.com/AndrewCooper/docker-svn.git
 buildbot_image_repo: https://github.com/AndrewCooper/docker-buildbot-master.git
+buildbot_worker_ntel_image_repo: https://github.com/AndrewCooper/docker-buildbot-worker-ntel.git
+buildbot_worker_ptxdist_image_repo: https://github.com/AndrewCooper/docker-buildbot-worker-ptxdist.git
